### PR TITLE
added one line note about status

### PR DESCRIPTION
**Pull request by himorin**
added at the top instead of at some middle point, for ease to be noticed when someone visited to the page.

### CODE REVIEW

**Comment by himorin** [comment]
(please ignore about the build failure. it's just I'm using my fork, and would not be an issue on merging PR - even if failed, this PR does not touch any spec file)

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+IMPORTANT NOTE: Development of this Timed Text Markup Language 3 (TTML3) 
+specification is currently suspended.
```
Comment (nigelmegitt) [inline README.md:2]:
```suggestion
**IMPORTANT: Development of Timed Text Markup Language 3 (TTML3) is currently suspended.**

To raise issues against TTML2 please visit [w3c/ttml2 issues](https://github.com/w3c/ttml2/issues).

This repository serves to hold issues against TTML whose resolution requires a substantive change that is deemed too significant to make in previous versions of TTML.
```

**Review by nigelmegitt — CHANGES_REQUESTED** [review]
I think we can be even more helpful, suggestion provided.

**Review by nigelmegitt — APPROVED** [review]
LGTM

**Pull request merged**